apps/test-suite/tests/Contacts.js

```diff
@@ -485,6 +485,33 @@ export async function test({ describe, it, xdescribe, jasmine, expect, afterAll
       expect(result[Contacts.Fields.FirstName]).toEqual('Andy');
     });
 
+    it('Contacts.updateContactAsync() and toggle isFavorite', async () => {
+      const contacts = await Contacts.getContactsAsync({
+        fields: [Contacts.Fields.IsFavorite],
+      });
+      const favoriteContact = contacts.data.find((contact) => contact.isFavorite);
+      expect(favoriteContact).toBeDefined();
+      expect(typeof favoriteContact.isFavorite).toBe('boolean');
+      await Contacts.updateContactAsync({
+        id: favoriteContact.id,
+        [Contacts.Fields.IsFavorite]: false,
+      });
+      let modifiedContact = await Contacts.getContactByIdAsync(favoriteContact.id, [
+        Contacts.Fields.IsFavorite,
+      ]);
+      expect(typeof modifiedContact.isFavorite).toBe('boolean');
+      expect(modifiedContact.isFavorite).toBe(false);
+      await Contacts.updateContactAsync({
+        id: favoriteContact.id,
+        [Contacts.Fields.IsFavorite]: true,
+      });
+      modifiedContact = await Contacts.getContactByIdAsync(favoriteContact.id, [
+        Contacts.Fields.IsFavorite,
+      ]);
+      expect(typeof modifiedContact.isFavorite).toBe('boolean');
+      expect(modifiedContact.isFavorite).toBe(true);
```

Comment:
```suggestion
      const favoriteContact = contacts.data.find((contact) => contact.isFavorite);
      expect(favoriteContact).toBeDefined();
      expect(favoriteContact.isFavorite).toBe(true);
      await Contacts.updateContactAsync({
        id: favoriteContact.id,
        [Contacts.Fields.IsFavorite]: false,
      });
      let modifiedContact = await Contacts.getContactByIdAsync(favoriteContact.id, [
        Contacts.Fields.IsFavorite,
      ]);
      expect(modifiedContact.isFavorite).toBe(false);
```

we don't need to toggle back and forth, or do we? If other test depends on the output of this one, that's a smell.